src/transactionComputer.ts

```diff
@@ -79,5 +104,17 @@ export class TransactionComputer {
         return value && value.length ? Buffer.from(value).toString("base64") : undefined;
     }
 
-    // TODO: add missing functions wrt. specs
+    private ensureFields(transaction: ITransaction) {
```

Comment:
Can be a function that covers more constraints on the purpose of validating the transaction prior signing. E.g. options vs. version. Can be done in another PR.

Reply:
Will be done in the next PR.